src/coreComponents/physicsSolvers/multiphysics/SinglePhasePoromechanicsSolver.cpp

```diff
@@ -67,30 +64,19 @@ PoroelasticSolver::PoroelasticSolver( const string & name,
     setInputFlag( InputFlags::REQUIRED ).
     setDescription( "Coupling method. Valid options:\n* " + EnumStrings< CouplingTypeOption >::concat( "\n* " ) );
 
-  m_linearSolverParameters.get().mgr.strategy = LinearSolverParameters::MGR::StrategyType::singlePhasePoroelastic;
+  m_linearSolverParameters.get().mgr.strategy = LinearSolverParameters::MGR::StrategyType::singlePhasePoromechanics;
   m_linearSolverParameters.get().mgr.separateComponents = true;
   m_linearSolverParameters.get().mgr.displacementFieldName = keys::TotalDisplacement;
   m_linearSolverParameters.get().dofsPerNode = 3;
 }
 
-void PoroelasticSolver::registerDataOnMesh( Group & meshBodies )
+void SinglePhasePoromechanicsSolver::registerDataOnMesh( Group & meshBodies )
```

Comment:
This override can now be removed entirely

Reply:
Done.